src/backend/executor/cypher_set.c

```diff
@@ -160,7 +172,8 @@ static HeapTuple update_entity_tuple(ResultRelInfo *resultRelInfo,
                                 (update_indexes == TU_Summarizing));
         }
 
-        ExecCloseIndices(resultRelInfo);
+        if (close_indices)
```

Comment:
Our coding standard requires that all if/else blocks must have opening and closing braces.